superset-frontend/src/components/Label/Label.stories.tsx

```diff
@@ -25,9 +25,10 @@ export default {
   title: 'Label',
   component: Label,
   decorators: [withKnobs],
+  excludeStories: ['bsStyleKnob'],
 };
 
-const bsStyleKnob = {
+export const bsStyleKnob = {
```

Comment:
maybe we should move this to a fixtures file that the stories and the test can both import from? It's a bit weird to import from a stories file in the test file, and it would also mean you don't need to tell storybook that this isn't a story too

Reply:
In this case (exporting the knobs), I agree it's a little weird, but I really like importing the actual _stories_ in tests, so I figured this could come along for the ride. Mounting the story exports helps make sure your tests cover the use cases of the component properly, and means that storybook edits (i.e. when adding new features/variants to a component) might cause test failures, which I think is a _good_ thing.

All that being said, I'll try to find a good way around the weird knob export/exclusion (probably in another PR). It is a little screwy.